### ISSUE TITLE
Urganite does not retreat into their shell

### ISSUE THREAD

**Issue by icariumkupo** [EdenServer/community#1514]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

ver 30181205_0

Urganites should retreat into their shell when taking physical damage. Resulting in a strong regen effect and -75% physical damage taken. It also makes them use only Venom Shell as their TP move. 

https://ffxiclopedia.fandom.com/wiki/Category:Uragnites

**Comment by Sulfuricsixx** [comment]
I can also confirm that they do not retreat into their shells and gain the 75% damage reduction. However, the spell is still able to be learned. They will use the TP move still even without retreating into their shells. Unfortunately it can take some time. I killed almost 15-20 until I saw the TP move used. Learned the skill at about the 30-35th kill. The joys of Learning Blue Magic! 

![Sulfuricsixx_2019 04 09_202137](https://user-images.githubusercontent.com/49461776/55843232-3396dd80-5b05-11e9-9242-228fce3cdb98.png)
